docs/recipes/zipkin.yaml

```diff
@@ -0,0 +1,40 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: zipkin
+  labels:
+    app.kubernetes.io/name: 'zipkin'
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: zipkin
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: 'zipkin'
+    spec:
+      containers:
+      - name: zipkin
+        image: openzipkin/zipkin
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 9411
+...
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: zipkin
+  labels:
+    app.kubernetes.io/name: 'zipkin'
+spec:
+  selector:
+    app.kubernetes.io/name: zipkin
+  type: ClusterIP
+  ports:
+  - name: http
+    protocol: TCP
+    port: 9411
+    targetPort: 9411
```

Comment:
Note, I didn't take the step of actually deploying zipkin for you, because it requires databases.

It really seems like there isn't a *standard* or *official* route for zipkin. We might want to pick Jaeger as the recommended dtrace package. They seem to have lot more options and documentation for deployment.